### PR TITLE
Added containsSelectedFieldOfStudy method

### PR DESCRIPTION
**Pull request by jsung23**
Edited buildConditionString method to accomodate <value> IN <multiple columns>. I looked through the school-related tables and don't see any other use case that'd use this method, so I think the custom-ness of it is fine.

### CODE REVIEW

**Comment by jsung23** [comment]
Fixed containsSelectedFieldOfStudy method to use REVERSE_IN condition type. fixes #17  . fixes #12 . fixes #13

**Comment by jsung23** [comment]
Also fixes #18

**Comment by jsung23** [comment]
I'm not sure why that orig file was created. For keeping fixes in separate
branches, don't they all become part of the same pull request as I merge
each branch into local/master and push to my origin/master? Or is the idea
that I would pull request 1 fix at a time and not push other fixes to
origin/master until that first pull request has been accepted?

**Comment by geomcmaster** [comment]
Don't push them to origin master! Sorry I must have confused you. If you have a branch named "branch_1" you would push to origin branch_1.

**Comment by jsung23** [comment]
O got it!

**Comment by jsung23** [comment]
And I'd first pull request to merge origin/branch_1 with upstream/master,
then after that goes through, merge with origin/master? Or that doesn't
matter?

**Comment by geomcmaster** [comment]
The direction it should flow is from the main repository. So after the pull request is accepted if you want to update your own master branch, pull upstream master into local master then push origin master.

File: CollegeMatchWebApp/src/main/java/CondType.java [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+ * NE
+ * 		<> ?
+ * LIKE
+ * 		LIKE ?
+ * IN
+ * 		IN ?
+ * 
+ * OR_GROUP
+ * 		(cond1 OR cond2 OR ...)
+ *
```
Comment (geomcmaster) [inline CollegeMatchWebApp/src/main/java/CondType.java]: Add a comment for REVERSE_IN

**Pull request closed without merging**